Cargo.toml

```diff
@@ -13,4 +13,5 @@ log = "^0.4.6"
 schemars = { version = "0.8", features = ["chrono", "uuid08"] }
 serde = { version = "^1.0", features = ["derive"] }
 serde_derive = "1.0"
+sqlx = "0.5"
```

Comment:
I'd recommend doing`default-features = false` so you don't pull in the macros and migrate crates and such.